CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+Changelog
+=========
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to
+[Semantic Versioning](https://semver.org/).
+
+But we group the changes as follows:
+
+* *Breaking Changes* when a change requires manual activity at update.
+* *Bug Fixes* when we fix broken functionality.
+* *New Features* for added functionality.
+* *Improvements* for other changed parts.
+
+
+Table of Contents
+-----------------
+
+[TOC]
+
+
+[Unreleased]
+------------
+
+### Breaking Changes
+
+### Bug Fixes
+
+### New Features
+
+### Improvements
+
+
+--------------------------------------------------------------------------------
+
+
+[1.6.0] (2019-11-06)
+--------------------
+
+### Improvements
+
+* Improve `diff` command with better messages and a summary at the end.
+
+
+--------------------------------------------------------------------------------
```

Comment:
There should be only one separator that separates unreleased from the most recent release.